app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -233,11 +233,7 @@ class DescriptionEditFragment : Fragment() {
                         val baseRevId = mwQueryResponse.query()!!.firstPage()!!.revisions()[0].revId
                         text = updateDescriptionInArticle(text, binding.fragmentDescriptionEditView.description!!)
                         ServiceFactory.get(wikiSite).postEditSubmit(pageTitle.prefixedText, "0", null,
-                                when (action) {
```

Comment:
@sharvaniharan   If you look here, this block does not contain the check for `if (invokeSource == InvokeSource.SUGGESTED_EDITS || invokeSource == InvokeSource.FEED)`, as the others do.

Reply:
ah! ok thank you!